src/main/java/Wolox/training/models/Book.java

```diff
@@ -9,37 +11,47 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private int id;
 
-    @Column
+    @Column(name = "genre")
```

Comment:
by default the database should created the columns with the attribute name, so unless you want a different column name there no need to specify the name in the annotation